commit cb2194a8d2ce83a5e3a8ae5c24c9bbf1c4b8b7da (refs/changes/69/269/1)
Author: Marisa Emerson <mje@uq.edu.au>
Date:   2016-08-11T14:52:58+10:00 (3 years, 2 months ago)
    
    allow more than one zone to use a wildcard as the networks setting

diff --git a/lib/net-filter.js b/lib/net-filter.js
index 52a3a21..df51975 100644
--- a/lib/net-filter.js
+++ b/lib/net-filter.js
@@ -69,11 +69,12 @@ NetFilter.prototype._transform = function (vm, enc, cb) {
 				}
 			}
 		}
-		for (i = 0; i < zones.length; ++i) {
-			z = self.config[zones[i]];
-			if (z.networks.indexOf('*') !== -1 &&
-			    nic.zones.length === 0)
-				nic.zones.push(zones[i]);
+		if (nic.zones.length === 0) {
+			for (i = 0; i < zones.length; ++i) {
+				z = self.config[zones[i]];
+				if (z.networks.indexOf('*') !== -1)
+					nic.zones.push(zones[i]);
+			}
 		}
 	});
 
